docs/src/TextArea.doc.js

```diff
@@ -166,6 +168,31 @@ function Example(props) {
   />,
 );
 
+card(
+  <Example
+    id="errorMessageExample"
+    name="Example: Error message"
+    description={`
+    A TextArea can display its own error message.
+    To use our errors, simply pass in an \`errorMessage\` when there is an error present and we will     handle the rest.`}
```

Comment:
can we replace "our" with "built in" or "default"?


also big space there: `we will handle the rest`

Reply:
we will handle the rest. could also be said as `TextArea handles the error display`

Reply:
I wasn't updating any of the existing content — we'll take a pass at this in future work when we overall the whole page